test_scripts/Defects/7_1/1619_AlertManeuer.lua

```diff
@@ -0,0 +1,76 @@
+--------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1619
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with valid AlertManeuver response (empty parameter "info")
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session started
+-- 3) App is registered
+-- 4) PTU
+-- 5) App is activated
+--
+-- Steps:
+-- 1) Send AlertManeuver mobile RPC from app with parameter softButtons = [], HMI responds with
+--    SUCCESS to Navigation.AlertManeuver request
```

Comment:
It would be better to split this step into 2 since there are 2 actions - 1st by App and 2nd by HMI